lib/yabeda/group.rb

```diff
@@ -9,6 +9,10 @@ class Group
 
     param :name
 
+    def default_tags
```

Comment:
Probably should be a different name since this doesn't include the default_tags of the global scope?